### PR TITLE
LR-170. Top menu

### PR DESCRIPTION
**Pull request by OKrain**
[Link to JIRA ticket](http://ssu-jira.softserveinc.com/browse/LR-170)

### Author.

- [x] I have added link to this PR to the JIRA
- [x] I have logged time in JIRA.

### Reviewers.

- [x] @sergiy17 
- [x] @anatoliiibrahimov 
- [x] @vovka

### Issue summary.

Add top menu

### Fix summary. 

Implement top menu with bootstrap

### Testing approach.

Included Devise test helpers. 
Added two tests to check the view.

### CODE REVIEW

File: spec/views/layouts/application.html.erb_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+RSpec.describe "layouts/application" do
+  it "home page for not sign in user" do
+    render
+
+    expect(rendered).to match %r{<a href="/">Home</a>}
+    expect(rendered).to match %r{<a href="/users/auth/facebook">Sign in with Facebook</a>}
```
Comment (vovka) [inline spec/views/layouts/application.html.erb_spec.rb]: Codacy Issue found: [Line is too long. [90/80]](https://www.codacy.com/app/vovka/video_quiz/file/2376070199/issues/source?bid=3205144&fileBranchId=3224532#l7)

File: spec/views/layouts/application.html.erb_spec.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  it "home page for sign in user" do
+    user = User.create!(email: "sample@example.org", name: "Sample")
+    sign_in user
+    render
+
+    expect(rendered).to match %r{<a href="/quizzes">Home</a>}
+    expect(rendered).to match %r{<a rel="nofollow" data-method="delete" href="/sign_out">Log out</a>}
```
Comment (vovka) [inline spec/views/layouts/application.html.erb_spec.rb]: Codacy Issue found: [Line is too long. [101/80]](https://www.codacy.com/app/vovka/video_quiz/file/2376070199/issues/source?bid=3205144&fileBranchId=3224532#l16)

Comment (vovka) [inline spec/views/layouts/application.html.erb_spec.rb]: Codacy Issue found: [Line is too long. [101/80]](https://www.codacy.com/app/vovka/video_quiz/file/2376208646/issues/source?bid=3205144&fileBranchId=3224532#l16)

File: spec/views/layouts/application.html.erb_spec.rb (hunk 1 above)
Comment (vovka) [inline spec/views/layouts/application.html.erb_spec.rb]: Codacy Issue found: [Line is too long. [90/80]](https://www.codacy.com/app/vovka/video_quiz/file/2376208646/issues/source?bid=3205144&fileBranchId=3224532#l7)

File: spec/views/layouts/application.html.erb_spec.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
```
Comment (brouhaha88) [inline spec/views/layouts/application.html.erb_spec.rb]: ignore

**Comment by vovka** [comment]
looks goo to me

File: spec/views/layouts/application.html.erb_spec.rb (hunk 2 above)
Comment (vovka) [inline spec/views/layouts/application.html.erb_spec.rb]: Codacy Issue found: [Line is too long. [101/80]](https://www.codacy.com/app/vovka/video_quiz/file/2380766984/issues/source?bid=3205144&fileBranchId=3224532#l16)

File: spec/views/layouts/application.html.erb_spec.rb (hunk 1 above)
Comment (vovka) [inline spec/views/layouts/application.html.erb_spec.rb]: Codacy Issue found: [Line is too long. [90/80]](https://www.codacy.com/app/vovka/video_quiz/file/2380766984/issues/source?bid=3205144&fileBranchId=3224532#l7)

**Pull request merged**